### PR TITLE
Board Game Night

### PR DESCRIPTION
**Pull request by benabraham**
to do:
- [x] check details for this year

### CODE REVIEW

File: pyconcz/templates/pages/board_game_night.html [hunk 1]
```diff
@@ -19,31 +19,29 @@ <h1>Board Game Night</h1>
...
+                    <li>If this is your first night at Board Game Night, you <strong class="text-uppercase">have</strong> to play. (Unless you don’t want to.)</li>
                     <li>You can bring your own board games.</li>
-                    <li>PyCon CZ <a href="{% url 'about_code' %}">CoC</a> applies to Board Game Night.</li>
+                    <li>PyCon&nbsp;CZ <a href="{% url 'about_code' %}">CoC</a> applies to Board Game Night.</li>
                 </ol>
             </div>
             <div class="col-md">
                 <h2>Also good to know</h2>
                 <ul>
-                    <li>We will bring some games (DiXit, Pandemic, Steampunk Rally, Vast, Tzolk'in, Ubongo, Roll for the Galaxy, Robo Rally, 5 Minute Dungeon, Smallworld, Patchwork, Exploding Kittens, Hanabi, Carcassonne and more) but you’re welcome to bring your own as well!</li>
```
Comment (encukou) [inline pyconcz/templates/pages/board_game_night.html]: I can bring the same games again this year. Probably some more, but I'll not be able to check until about May 14.

Comment (hroncok) [reply pyconcz/templates/pages/board_game_night.html]: Ok. Let's not list any and add them later?

Comment (benabraham) [reply pyconcz/templates/pages/board_game_night.html]: Let's publish what we know and add more later. We can even suggest making pull requests to this template with games ppl will bring 🤔😁

Comment (benabraham) [reply pyconcz/templates/pages/board_game_night.html]: @hroncok can we publish what we have and update later?

Comment (hroncok) [reply pyconcz/templates/pages/board_game_night.html]: I consider it silly to list 2 games (would rather not list anything now), but use your best judgement.

**Comment by benabraham** [comment]
OK basic info published. 
Rest of tasks in #171.

**Pull request merged**